### ISSUE TITLE
Issiu with in app icon

### ISSUE THREAD

**Issue by Slim08** [microsoft/terminal#18317]
### Windows Terminal version

1.21.3231.0

### Windows build number

10.0.19045.5198

### Steps to reproduce

Change default PNG resource with any std Windows ICO resource and you get one behavior. Add ",0" at the end for another behavior. Both together would be correct as a whole. Individually, both are wrong.

[Screenshots](https://postimg.cc/gallery/pP9SHw6)

### Expected Behavior

Correct usage of the icon resource for the given place it's supposed to be displayed.

### Actual Behavior

Wrong or no icon displayed in taskbar jump list or in app.

**Comment by zadjii-msft** [comment]
Screenshots from the link

![Image](https://github.com/user-attachments/assets/a5fe07a1-e804-4dc0-a974-e68dba091609)
![Image](https://github.com/user-attachments/assets/d8e76634-ced1-4a71-be61-e9a347ee58e5)

**Comment by DHowett** [comment]
Thanks for the report!

Yes, it looks like we need to make sure `foo.ico,0` works.

However: you appear to be using a custom shell.

Confirming that this works in the normal Windows 10 shell:

![Image](https://github.com/user-attachments/assets/0b21f5be-7cf3-4e26-a7b6-54c2804a3bcf)

If your custom shell does not support icon file references, file a bug on them. Thanks!

_Also, why are you placing icon files in system32?_

**Comment by Slim08** [comment]
> Yes, it looks like we need to make sure foo.ico,0 works.

Thanks.

> However: you appear to be using a custom shell.

I'm not 100% sure, but the taskbar context menu there could potentially be influenced by StartIsBack which I have installed, and which I'm not going to uninstall over this. Other than that, I'm using the default Aero them of Windows 10.

> Also, why are you placing icon files in system32?

Since Terminal is a UWP app and I can't place the icon resource inside its installation directory. Any serious reason I couldn't / shouldn't put it in system32?

**Comment by DHowett** [comment]
> Any serious reason I couldn't / shouldn't put it in system32?

It makes your settings less portable, and won't be backed up or (in the future) roamed elsewhere with the rest of your user data. You can put icons in the same folder as your settings and refer to them with `ms-appdata://` URIs, or place them in your user Documents folder which _will_ be synced or roamed.

You may want to report a bug to StartIsBack that they are handling jump list icon references incorrectly.

**Comment by Slim08** [comment]
Yeah, well, I don't do all that fancy roaming, syncing or whatever. If I need to, I reinstall my OS fresh with all my custom stuff already backed up on a USB manually. So I don't care about portability or whatever.

StartIsBack is a Windows 10 exclusive app and out of active development for years. There is nothing going to change that, with even Microsoft dropping support next year.
I'm still not 100% sure that the jump list even is StartIsBack's fault. Plus, 

> Yes, it looks like we need to make sure foo.ico,0 works.

Not only, StartIsBack is handling icon references incorrectly around here, right?

**Comment by DHowett** [comment]
> Not only, StartIsBack is handling icon references incorrectly around here, right?

Yes. That's why we've kept the bug open instead of closing it.

To be clear, there are two issues here:

- One in Terminal, for not supporting `foo.ico,0`
- One in StartIsBack _or whatever is providing your custom jump list UI_, for not supporting `foo.ico` without an index.

> There is nothing going to change that, with even Microsoft dropping support next year.

Ok. Yes, I would not expect Microsoft dropping support for Windows 10 to suddenly return StartIsBack to active development. That seems like a concern for the original author of StartIsBack.
